batch/batch/templates/batches.html

```diff
@@ -24,6 +25,14 @@ <h1>Batches</h1>
             {% endif %}
           </td>
           <td>{{ batch['state'] }}</td>
+          <td>
+            {% if not batch['complete'] %}
+            <form action="/batches/{{ batch['id'] }}/cancel" method="post">
+              <input type="hidden" name="_csrf" value="{{ token }}"/>
```

Comment:
Could you open an issue, to explore changing this to a header-specified token, or randomizing the name field.


https://security.stackexchange.com/questions/211352/does-owasp-recommend-to-include-a-csrf-token-in-a-header-or-to-use-it-as-a-param
  * Need to take care with logging in this case.

https://github.com/OWASP/CheatSheetSeries/blob/master/cheatsheets/Cross-Site_Request_Forgery_Prevention_Cheat_Sheet.md
  * To further enhance the security of this proposed design, consider randomizing the CSRF token parameter name and/or value for each request. Implementing this approach results in the generation of per-request tokens as opposed to per-session tokens.
  * doing both seems identical to implementing 2 CSRF tokens